src/components/dependencies/Component.js

```diff
@@ -239,6 +241,39 @@ const Component = React.memo(({
     complianceSummaryState.error,
   ])
 
+  const [mountRescoring, setMountRescoring] = React.useState(searchParamContext.get('rescoreArtefacts'))
```

Comment:
nit: Rather parse to boolean explicitly so that `mountRescoring` has a constant (and expected) type.